RecoEgamma/EgammaTools/python/egammaObjectModificationsInMiniAOD_cff.py

```diff
@@ -59,3 +59,68 @@ def setup_mva(val_pset,cat_pset,prod_name,mva_name):
           egamma_modifications[3].photon_config,
           pho_mva_prod_name,
           pho_spring15_50_nt.mvaSpring15NonTrigClassName+pho_spring15_50_nt.mvaTag)
+
+#############################################################
+# REGRESSION MODIFIERS
+#############################################################
+
+egamma_modifications.append(
+    cms.PSet( modifierName    = cms.string('EGExtraInfoModifierFromDB'),  
+              autoDetectBunchSpacing = cms.bool(True),
+              bunchSpacingTag = cms.InputTag("addPileupInfo:bunchSpacing"),
+              manualBunchSpacing = cms.int32(50),              
+              rhoCollection = cms.InputTag("fixedGridRhoFastjetAll"),
+              vertexCollection = cms.InputTag("offlinePrimaryVertices"),
+              electron_config = cms.PSet( sigmaIetaIphi = cms.InputTag('electronRegressionValueMapProducer:eleFull5x5SigmaIEtaIPhi'),
```

Comment:
wouldn't it make more sense for the electronRegressionValueMapProducer to produce a struct/class with all these variables?